clr-k8s-examples/kubeadm.yaml

```diff
@@ -11,6 +12,7 @@ systemReserved:
 kubeReserved:
   cpu: 500m
   memory: 256M
+cgroupDriver: systemd
```

Comment:
Are we making sure the runtime (crio/containerd) also has cgroup driver configured to be systemd?

Reply:
Just curious. Why is that we have to hard-code this in our yaml? Are you hitting issues without this that we have missed?

Reply:
This is the main WIP part.

Reply:
Ubuntu fails without it. I will plan to `sed` it in the `system_setup_ubuntu.sh` unless you want me to take another approach?

Reply:
this is distro specific configuration of the container runtime. Ideally kubeadm should do the right thing. Better keep it in distro script?

Reply:
Removed!